openstack/maia/templates/seed.yaml

```diff
@@ -0,0 +1,92 @@
+{{- if .Release.IsUpgrade }}
+apiVersion: "openstack.stable.sap.cc/v1"
+kind: "OpenstackSeed"
+metadata:
+  name: maia-seed
+  labels:
+    component: maia
+spec:
+  requires:
+#    - keystone-seed
+#    - monsoon3/domain-cc3test-seed
+#    - monsoon3/domain-ccadmin-seed
+
+  roles:
+  - monitoring_viewer
+  - monitoring_admin
+
+# TODO expose as OpenStack service
```

Comment:
Is it really intended to expose maia as an OpenStack service? I thought it's just going to be a "wrapper" for customer's prometheus (see [here](https://github.com/sapcc/maia#using-maia)) and Grafana.